src/components/accessWrappers/EnrollmentRoute.tsx

```diff
@@ -20,14 +21,15 @@ const EnrollmentRoute: React.FC<
   }>
 > = ({ permissions, redirectRoute, dataCollectionFeature, children }) => {
   // Use dashboard outlet context that gets set in EnrollmentDashboard
-  const { enrollment, enabledFeatures } = useEnrollmentDashboardContext();
+  const { enrollment } = useEnrollmentDashboardContext();
```

Comment:
I'm not _totally_ convinced on the benefit of moving `enabledFeatures` from the context to a new hook. It's not a bad approach but it does cause some more code churn. Just to confirm-- we're never using the enrollment dataCollectionFeatures _outside_ of the enrollment dashboard context, right? (I wouldn't expect that, since I think of them of features that may or may not be enable "on" the Enrollment Dashboard).


What are you seeing as the benefit of moving it into a hook? I don't mean to nit-pick, but it might be helpful to just talk through the options.

_Another_ approach could be for it just to be a simple helper function, which I think would be fine. We lose the `useMemo` from the hook, but it's not an expensive operation (we know that dataCollectionFeatures array is small).



util fn:
```
const { enrollment } = useEnrollmentDashboardContext(); // or could come from a prop

if (getEnrollmentFeature(enrollment, DataCollectionFeatureRole.CaseNote)) { ... }
```

vs keeping it in context:
```
const { getEnrollmentFeature } = useEnrollmentDashboardContext(); // need to consume context

if (getEnrollmentFeature(DataCollectionFeatureRole.CaseNote)) { ... }
```


vs hook:
```
const { enrollment } = useEnrollmentDashboardContext(); // or could come from a prop

const caseNotes = useEnrollmentDataCollectionFeature(enrollment, DataCollectionFeatureRole.CaseNote)
if (caseNotes) { ... }
```

Reply:
Oh, thanks for bringing this up. I like the middle option the best, where we have the context provide a `getEnrollmentFeature` function (whereas before it just used to provide the `enabledFeatures` list).

I don't think my original reasoning for this change was very interesting. It was something like: first I removed `enabledFeatures` because this list is now a property on the `enrollment` itself and doesn't need to be provided separately by the context. Later I was bothered by the repetition of the code to scan the list of enabled features looking for a particular one, so I wanted to factor that out into a separate place. There is no reason that separate place shouldn't be moving it right back into the enrollment context from whence it came, haha.

Reply:
Okay, yeah that makes sense- def agree its helpful to have a function or map to make it easier to access the feature you're looking for. That decision sounds good to me .